AUTHORS.txt

```diff
@@ -1,3 +1,5 @@
+Authors ordered by first contribution.
```

Comment:
This line was accidentally removed at some point. The `main` branch still has it.